### PR TITLE
add rule for Ze.tt

### PR DESCRIPTION
**Pull request by numismatika**
Youth magazine from Zeit Online

### CODE REVIEW

File: src/chrome/content/rules/Ze.tt.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!-- for more Zeit-Online rules  see Zeit.de.xml -->
```
Comment (J0WI) [inline src/chrome/content/rules/Ze.tt.xml]: Please also update the reference list there

**Comment by J0WI** [comment]
`Zeit.de.xml` has no test coverage, so tests are failing.

**Comment by numismatika** [comment]
just pushed a new iteration to see how much work for that general rewrite of the Zeit.de rule is required.
maybe we are lucky with a few more additional target hosts.

**Comment by numismatika** [comment]
@J0WI  we have coverage onthe Zeit.de domain again

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**